lib/update/index.js

```diff
@@ -0,0 +1,7 @@
+import _curry3 from '../_internal/_curry3'
+
+export default _curry3(function update(idx, value, list) {
+  var result = list.slice()
+  result[idx - 1] = value
```

Comment:
why `index - 1`?

Reply:
hmm, you're right. I thought that this method assume count from `1`, but I've mistaken